VariablesHomework/VariablesHomework.cs

```diff
@@ -16,130 +22,131 @@ public void Declaration()
             Assert.IsTrue(character == 'z'); // Make this pass without changing this statement
         }
 
-        [TestMethod]
+       [TestMethod]
         public void Assignment()
-        {
-            int i = 0;
-            Assert.IsTrue(i == 25); // Make this pass without changing this statement (add or modify a statement above to make it pass)
-
+                {
+        int i = 25;
+        Assert.IsTrue(i == 25); // Make this pass without changing this statement (add or modify a statement above to make it pass)
+            i = 100;
             Assert.IsTrue(i == 100); // Then make this pass without changing this statement
 
-            float f = 0.0f;
-            Assert.IsTrue(f == 3.5); // Make this pass without changing this statement (except you might need to add something)
-
+            float f = 3.5f;
+        Assert.IsTrue(f == 3.5); // Make this pass without changing this statement (except you might need to add something)
+            f = 99.9f;
             Assert.IsTrue(f == 99.9); // Then make this pass without changing this statement (except you might need to add something)
```

Comment:
This line is failing because of a rounding error. Your assignment is correct, but you'll actually need to change the assert slightly (which is why the comment says "you might need to add something").
In the assert, you'll need to add a suffix to make the compiler treat `99.9` as the correct data type.